vllm/model_executor/models/gemma.py

```diff
@@ -325,15 +325,23 @@ def load_weights(self,
                 if shard_name not in name:
                     continue
                 name = name.replace(shard_name, param_name)
+                # Skip loading extra bias for GPTQ models.
+                if name.endswith(".bias") and name not in params_dict:
+                    continue
                 param = params_dict[name]
                 weight_loader = param.weight_loader
                 weight_loader(param, loaded_weight, shard_id)
                 break
             else:
+                # Skip loading extra bias for GPTQ models.
+                if name.endswith(".bias") and name not in params_dict:
```

Comment:
Could you merge this check with the one above and place it outside the for loop?

Reply:
I copy them from https://github.com/vllm-project/vllm/blob/main/vllm/model_executor/models/llama.py#L377-L379 and https://github.com/vllm-project/vllm/blob/main/vllm/model_executor/models/llama.py#L385-L387.
Modifying the `for-else` structure may cause excessive modifications. I recommend reusing existing code.

Reply:
@TechxGenus I think we can do this optimization on `Gemma` model file. Just put this check onto `for (param_name, shard_name, shard_id) in stacked_params_mapping:` without changing `for-else` code part.